common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -174,13 +186,15 @@ public ManagedBuffer getBlockData(
       String execId,
       int shuffleId,
       int mapId,
-      int reduceId) {
+      int reduceId,
+      int stageAttemptId) {
```

Comment:
shall we call it `shuffleGenerationId` which is the name we use in the [caller side](https://github.com/apache/spark/pull/24892/files#diff-069b5ce9dd07165f0c185249cf4a04cbR314)?

Reply:
Thanks, done in 5a4b8b8.